x-pack/plugins/observability_ai_assistant/public/components/chat/welcome_message_knowledge_base_setup_error_panel.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiCode,
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiDescriptionListDescription,
+  EuiLink,
+  EuiSpacer,
+  EuiIcon,
+  EuiText,
+  EuiHorizontalRule,
+  EuiPanel,
+} from '@elastic/eui';
+import { css } from '@emotion/css';
+import { useKibana } from '../../hooks/use_kibana';
+import type { UseKnowledgeBaseResult } from '../../hooks/use_knowledge_base';
+
+const panelContainerClassName = css`
+  width: 330px;
+`;
+
+export function WelcomeMessageKnowledgeBaseSetupErrorPanel({
+  knowledgeBase,
+  onRetryInstall,
+}: {
+  knowledgeBase: UseKnowledgeBaseResult;
+  onRetryInstall: () => void;
+}) {
+  const { http } = useKibana().services;
+
+  const modelName = knowledgeBase.status.value?.model_name;
+
+  return (
+    <div
+      className={panelContainerClassName}
+      data-test-subj="observabilityAiAssistantWelcomeMessageKnowledgeBaseSetupErrorPanel"
+    >
+      <EuiPanel hasBorder={false} hasShadow={false} paddingSize="m">
+        <EuiDescriptionList>
+          <EuiDescriptionListTitle>
+            {i18n.translate(
+              'xpack.observabilityAiAssistant.welcomeMessage.issuesDescriptionListTitleLabel',
+              { defaultMessage: 'Issues' }
+            )}
+          </EuiDescriptionListTitle>
+
+          <EuiSpacer size="s" />
+
+          <EuiDescriptionListDescription>
+            <ul>
+              {!knowledgeBase.status.value?.deployment_state ? (
```

Comment:
Maybe it would make sense to unpack `knowledgeBase.status.value` earlier in the file to keep these conditions a bit shorter to read and avoid the repetition?